src/applications/facility-locator/components/SearchControls.jsx

```diff
@@ -15,20 +15,32 @@ import classNames from 'classnames';
 
 class SearchControls extends Component {
   handleQueryChange = e => {
-    this.props.onChange({ searchString: e.target.value });
+    this.props.onChange({ searchString: e.target.value.trim() });
```

Comment:
This change prevents the user from entering a leading or trailing space, and guarantees that `handleLocationBlur` will trigger a redux state change.